src/next.ts

```diff
@@ -45,12 +46,14 @@ export type Queue<Payload> = Omit<
 export function Queue<Payload>(
   route: string,
   handler: QuirrelJobHandler<Payload>,
-  defaultJobOptions?: DefaultJobOptions
+  defaultJobOptions?: DefaultJobOptions,
+  logger?: CustomLogger<Payload>
```

Comment:
I think we should combine these two arguments into one and make it a "options" argument. I'll make some commits to change it to how I imagine the API. Not sure when I get to it, probably within the next weeks.

Reply:
I outlined what I'm talking about in https://github.com/quirrel-dev/quirrel/pull/1039/commits/9d9ab0e82accb5a6215ce7ae293a765e73003b1a. How do you like that?

Reply:
I like that a lot! Gtg from my side, of course!